<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="client.qrc">
    <normaloff>:/icons/red.png</normaloff>:/icons/red.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ChatTab">
      <attribute name="title">
       <string comment="TitleChat">Chat</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="ChatTextView">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ChatTextEntry" name="EntryTextEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="UsersTab">
      <attribute name="title">
       <string comment="TitleUsers">Users</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="UsersTreeWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="autoScroll">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>100</number>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SettingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="ShowNameCheckBox">
         <property name="text">
          <string comment="Showscore">Show names and score</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="PlaySoundCheckBox">
         <property name="text">
          <string comment="PlaySound">Play sound</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="PlayMusicCheckBox">
         <property name="text">
          <string comment="PlayMusic">Play music</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatTextEntry</class>
   <extends>QPlainTextEdit</extends>
   <header>chattextentry.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections/>
</ui>
